cobalt/BUILD.gn

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 group("gn_all") {
-  # TODO(b/371589344): Fix android build configs.
-  deps = [ "//starboard($starboard_toolchain)" ]
+  if (is_cobalt) {
```

Comment:
If we're adding a condition here, shouldn't it be removed in the toplevel BUILD.gn file?